LoopKitUI/Extensions/UINavigationController.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  UINavigationController.swift
+//  LoopKitUI
+//
+//  Created by Nathaniel Hamming on 2020-01-27.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+extension UINavigationController {
+    public func groupedTableViewHiddenNavBarStyle() {
+        self.navigationBar.setBackgroundImage(UIImage(), for: .default)
```

Comment:
I don't understand what this is doing; setting empty images for the navigation bar background and shadowImage.

Reply:
This is making the navigation bar transparent, such that the title floats in the UI to match the designs.